templates/ruby/lib/container/client.rb.twig

```diff
@@ -88,6 +92,27 @@ module {{spec.title | caseUcfirst}}
 
             return JSON.parse(response.body);
         end
+        
+        def encodeFormData(value, key=nil)
+            case value
+            when Hash  then value.map { |k,v| encodeFormData(v,k) }.join
+            when Array then value.map { |v| encodeFormData(v, "#{key}[]") }.join
+            when nil   then ''
+            else
+                post_body = []
+                if(key.to_s == "file")
```

Comment:
Yep the hardcoded key might be a bit problematic, cloud functions will also need file upload under a different key name. In other languages we accept a file instance and that way we can detect its type. Maybe the createFile method should expect a File.read(value) instance and not a string.

Another thing we need to do, is to verify the code examples for file uploads is correct.

Reply:
My bad, did i forget to mention the hard coded file key. I will investigate further.